kbase-extension/scss/partials/_dropzoneDz.scss

```diff
@@ -78,8 +79,8 @@
 
 .kb-dropzone {
   border: 2px dashed use_color('mid-blue') !important;
-  margin-bottom: 5px;
-  max-height: 150px;
+  // margin-bottom: 5px;
```

Comment:
You can just delete these. I don't think they need to come back.

Reply:
Roger that.